Fabric.Identity.API/scripts/tests/testAzure.config

```diff
@@ -12,14 +12,14 @@
       </replyUrls>
```

Comment:
I believe that this testAzure.config file is expected to be blank, and it gets populated by different tests.